xml/System.Web.Security/MembershipProvider.xml

```diff
@@ -31,8 +31,12 @@
     <remarks>
       <format type="text/markdown"><![CDATA[  
   
-## Remarks  
- ASP.NET membership is designed to enable you to easily use a number of different membership providers for your ASP.NET applications. You can use the supplied membership providers that are included with the .NET Framework, or you can implement your own provider.  
+## Remarks
+
+> [!NOTE]
+> In-box ASP.NET membership providers have been superseded by [ASP.NET Identity](/aspnet/identity/overview/getting-started/introduction-to-aspnet-identity). Please update applications to use the ASP.NET Identity platform instead of the in-box membership providers.
+
+ASP.NET membership is designed to enable you to easily use a number of different membership providers for your ASP.NET applications. You can use the supplied membership providers that are included with .NET Framework, or you can implement your own provider.  
```

Comment:
Let's agree on the text. `In-box` is jargon and won't MT
https://github.com/dotnet/AspNetDocs/pull/845/files